clusterloader2/pkg/prometheus/experimental.go

```diff
@@ -139,7 +139,9 @@ func (pc *PrometheusController) trySnapshotPrometheusDisk(pdName, snapshotName,
 	if project == "" {
 		// This should never happen when run from kubetest with a GCE/GKE Kubernetes
 		// provider - kubetest always propagates PROJECT env var in such situations.
-		return fmt.Errorf("unknown project - please set --experimental-snapshot-project flag")
+		err := fmt.Errorf("unknown project - please set --experimental-snapshot-project flag")
+		klog.Error(err)
```

Comment:
we shouldn't klog error and return it - it often leads to multiple logs about the same thing...

Reply:
I have not decided yet if I want to merge it, I am using it to debug jobs.
But right now we do not have any logs.

Reply:
But yes - there is a better place to log it, I will push a fix